plyer/platforms/android/gps.py

```diff
@@ -29,6 +29,17 @@ def onLocationChanged(self, location):
             altitude=location.getAltitude(),
             accuracy=location.getAccuracy())
 
+    @java_method('(Ljava/util/List;)V', name='onLocationChanged')
```

Comment:
Would be useful to have doc strings explaining the difference between `onLocationChanged` and `onLocationChangedList`? It's not obvious here...